schemas/com.snowplowanalytics.snowplow.storage/snowflake_config/jsonschema/2-0-0

```diff
@@ -0,0 +1,120 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Snowplow Snowflake storage configuration",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow.storage",
+        "name": "snowflake_config",
+        "format": "jsonschema",
+        "version": "2-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "name": {
```

Comment:
Can we add human-readable descriptions please? I wasn't pedantic enough when designing first versions.